templates/waggle-dance-federation.yml.tmpl

```diff
@@ -1,6 +1,6 @@
 primary-meta-store:
   access-control-type: ${primary_metastore_access_type}
-  database-prefix: ''
+  database-prefix: ${primary_metastore_database_prefix}
```

Comment:
you might want to add the `_`
That's what we do for the others e.g.: https://github.com/ExpediaGroup/apiary-federation/blob/b0185a1a19c178e3f28b5c5de682b6fc079061f4/templates/waggle-dance-federation-local.yml.tmpl#L4

Reply:
agreeed